bassdrive-archive.js

```diff
@@ -214,5 +221,5 @@ spiderForMp3('http://archives.bassdrivearchive.com').then(function(){
       });
     }
   });
-}, {concurrency: 1}).all();
+}, {concurrency: argv.c).all();
```

Comment:
Should be `argv.c}` instead of `argv.c)`.  I will fix this in the merge.  Just FYI.